images/core.proto

```diff
@@ -52,7 +52,7 @@ message task_core_entry {
 	//optional int32		tty_nr		= 16;
 	//optional int32		tty_pgrp	= 17;
 
-	optional int32			child_subreaper	= 18;
+	optional bool			child_subreaper	= 18;
```

Comment:
I think it will work fine, but technically the documentation says:
```
PR_GET_CHILD_SUBREAPER (since Linux 3.4)
    Return the "child subreaper" setting of the caller, in the location pointed to by (int *) arg2.
```

Reply:
in [protobuf-c.h](https://github.com/racker/protobuf-c/blob/0dd05d11684fcd476088119ed3331ee7de0d321d/src/google/protobuf-c/protobuf-c.h#L88) bool is
```C
/** Boolean type. */
typedef int protobuf_c_boolean;
```
but using a boolean is better because it indicates that the value should be 0 or 1.

Reply:
> I'm okay with bool, but I used int since the documentation says:

If you get a value which isn't 0 or 1, you have to fail, because you don't know what it means...